src/defaultCredentials.ts

```diff
@@ -0,0 +1,89 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import * as api from "./api";
+import { configstore } from "./configstore";
+import * as logger from "./logger";
+
+interface RefreshTokenCredential {
+  client_id: string;
+  client_secret: string;
+  refresh_token: string;
```

Comment:
I would very strongly suggest using camelCase for consistency with the rest of the codebase. If underscores are required, please silence the warning with a `eslint-disable-line` to acknowledge that they are required in this form.

Reply:
This is not a choice, we have to have this exact format for it to be a valid credential.

Reply:
Got it. I think that's an even stronger case for adding the comments then - you don't want someone like me coming along and "cleaning these up" and causing it to break 🙂